console/console-init/ui/src/components/TypeAheadSelect/TypeAhead.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import {
+  Select,
+  SelectVariant,
+  SelectProps,
+  SelectOptionObject
+} from "@patternfly/react-core";
+import { placeholder } from "@babel/types";
+
+interface ITypeAheadSelectProps {
```

Comment:
as i suggested earlier we should use component interface so that we can avoid to create same properties again and can avoid to mismatch of types. here you can extends interface SelectProps. just create properties those are not available. and don't forget to export each and every interfaces so that we can reuse somewhere else.
interface ITypeAheadSelectProps extends SelectProps{
  //add new properties 
}